charts/trento-server/charts/rabbitmq/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: rabbitmq
+description: RabbitMQ is an open source general-purpose message broker that is designed for consistent, highly-available messaging scenarios (both synchronous and asynchronous).
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 2.6.0-dev1
```

Comment:
nit:
Maybe the version should be 1.0.0?
At the end, it is the 1st version of this subchart.
The 2.6.0 coming from `web` doesn't look appropriate